harness/determined/common/streams/__main__.py

```diff
@@ -1,8 +1,6 @@
 from determined.common import streams
-
```

Comment:
don't include changes from the harness code in your PRs.

Since three of us are working on a branch, this kind of PR overlap over noise will make it really hard to maintain a clean feature branch (which will require a lot of rebasing).